packages/shorebird_code_push_client/lib/src/code_push_client.dart

```diff
@@ -213,6 +213,20 @@ class CodePushClient {
     return Release.fromJson(body);
   }
 
+  /// Remove [userId] as a collaborator from [appId].
+  Future<void> deleteAppCollaborator({
+    required String appId,
+    required int userId,
+  }) async {
+    final response = await _httpClient.delete(
+      Uri.parse('$hostedUri/api/v1/apps/$appId/collaborators/$userId'),
```

Comment:
This feels like a helper on our code at some point, e.g.
endpoint(1, 'apps/$appId/collaborators/$userId')

🤷

Reply:
Yup I can add that in a subsequent PR 👍